sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -846,6 +857,15 @@ abstract class ToTimestamp
               case NonFatal(_) => null
             }
           }
+        case LongType =>
+          val input = t.asInstanceOf[Long]
+          if ( minRange < input && input < maxRange ) {
```

Comment:
Just in case, why do you exclude min and max values?

Reply:
Yes, min and max should also be considered as correct input.Thanks for your advise.